### DISCUSSION TITLE
Space key can not activate the "Quote Reply" button on issue page

### DISCUSSION THREAD

**Discussion by gdh1995** [community/community#39761] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Common HTML `<button>`s can be activated by <kbd>Space</kbd> and <kbd>Enter</kbd> keys, and most buttons on GitHub do support this usage.

But, today I find that `Quote Reply` buttons on a issue detail page will check `.isTrusted` of a browser-simulated click `MouseEvent` and then do nothing on `Space`/`Enter`.

I know we may select text using mouse and press <kbd>R</kbd> to quote it quicker, but this issue is still existing. And a patch for it sounds good, isn't it?

This also affects keyboard-shortcut web-extensions which hook keyboard inputs and simulate click events, including my `Vimium C` - as reported in https://github.com/gdh1995/vimium-c/issues/804 .

**Comment by ossanna16** [comment]
Hi there @gdh1995 and welcome to our community! Thank you for reporting this!
